src/main/java/nextstep/qna/domain/DeleteHistory.java

```diff
@@ -26,10 +26,24 @@ public DeleteHistory(ContentType contentType, Long contentId, NsUser deletedBy,
         this.createdDate = createdDate;
     }
 
+    public static DeleteHistory fromQuestion(Long questionId, NsUser writer) {
+        DeleteHistory deleteHistory = new DeleteHistory(ContentType.QUESTION, questionId, writer, LocalDateTime.now());
+        return deleteHistory;
```

Comment:
```suggestion
        return new DeleteHistory(ContentType.QUESTION, questionId, writer, LocalDateTime.now());
```
이 정도라면 지역변수없이 바로 리턴하는 것도 좋습니다.